draft-bormann-jsonpath-iregexp.md

```diff
@@ -78,17 +78,22 @@ Introduction        {#intro}
 
 The present specification defines an interoperable regular expression flavor, I-Regexp.
 
-This document uses the abbreviation "regexp" for regular expression.
+This document uses the abbreviation "regexp" for what is usually
+called regular expression in programming.
```

Comment:
It would be slightly more idiomatic to say "what are usually called regular expressions"